os/net/link-stats.c

```diff
@@ -147,6 +147,7 @@ link_stats_packet_sent(const linkaddr_t *lladdr, int status, int numtx)
     /* Add the neighbor */
     stats = nbr_table_add_lladdr(link_stats, lladdr, NBR_TABLE_REASON_LINK_STATS, NULL);
     if(stats != NULL) {
+      memset(stats, 0, sizeof(*stats));
```

Comment:
the two `memset` should not be necessary, as this is already done by `nbr_table`. Or am I missing something?

Reply:
True, the nbr table code does memset already.